src/cf.data.pre

```diff
@@ -2871,6 +2871,11 @@ DOC_START
 	Processing proceeds in the order specified, and stops at first fully
 	matching line.
 
+	The ipaddr may be one of the following labels (instead of a literal IP address):
+	    transparent: Sets the outgoing IP address to the same IP as the client's connection to the proxy.
```

Comment:
@yadij, to me, "transparent" feels too overloaded/abused of a term to be a good choice in this particular context, especially since the new feature does _not_ select the source IP address of the client TCP connection (something some admins will probably expect if we use "transparent"). I can suggest "use-client-tcp-destination-address", but that is fairly complex/long. We could (re)use "client-dst" or "use-client-dst", but that might be too cryptic? I hope you can suggest an even better name (or pick one of these three alternatives).

Reply:
Coming up with a new name for this must have been the hardest part of the entire PR so far 😂 
I've asked both friends and GPT models for suggestions, and considered everything from "passthrough" to "match-server-ip" and just "dynamic". However, the most precise way to refer to this functionality is by the TCP destination of the client's connection to Squid. With that I settled for `match_client_tcp_dst`, but I'm still open to changing it.